### PR TITLE
fix/deps

### PR DESCRIPTION
**Pull request by adekbadek**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

- move some packages from dev-dependencies to regular dependencies
- remove `@wordpress/blocks` - not used here

### How to test the changes in this Pull Request:

Nothing should change, a smoke test is enough.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,13 +22,8 @@
     "@typescript-eslint/eslint-plugin": "^5.3.0",
     "@wordpress/api-fetch": "^5.2.4",
     "@wordpress/base-styles": "^4.0.0",
```
Comment (noahtallen) [inline package.json:24]: Usually base-styles and api-fetch are also normal dependencies, unless something is different about their use here

Comment (adekbadek) [reply package.json]: Good catch, updated in e45395de55831a59bdb378f6462e86e4a3387d51

**Review by noahtallen — APPROVED** [review]
Seems like a good idea to me :)

**Comment by adekbadek** [comment]
I thought it's something wrong with this PR so I've rebased. But the culprit was on `master`. Awaiting https://github.com/Automattic/newspack-plugin/pull/1271 so the CI can pass here.

**Comment by dkoo** [comment]
@dependabot rebase

**Review by miguelpeixe — APPROVED** [review]
`npm ci && npm run build` is running without issues, CI tasks are passing and there's no apparent issue navigating through the wizards.

**Comment by dkoo** [comment]
@noahtallen @adekbadek all should be passing now.

**Pull request merged**